qe_metrics/manifests/services.example.yaml

```diff
@@ -0,0 +1,9 @@
+# Just an example of a couple of services.
+temp_service:
+  blocker: "some query to get the blocker issues"
+  critical-blocker: "some query to get the critical blocker issues"
+  customer-escaped: "some query to get the customer escaped issues"
```

Comment:
The query for escaped bugs is the same for all products 
"SFDC Cases Counter" > 0 and type = Bug and project = <Poject name> and created >= -14d
maybe we can build it with product name ?

Reply:
didn't we say that it is not bound to product name and different products can have either one or more jira projects?